docs/user/backends.md

```diff
@@ -73,6 +78,33 @@ healthy.
 SERVICE_CHECK_TTL=30s
 ```
 
+### Consul Docker Check
+
+This feature lets you register Docker checks with Consul, so your checks can
+run within existing containers instead of from the Consul agent.
+
+```bash
+SERVICE_DOCKER_CHECK_SCRIPT=/path/to/container/check
```

Comment:
My expectation as a user would be that these checks would be registered using `SERVICE_CHECK_DOCKER`, not `SERVICE_DOCKER_CHECK_SCRIPT`. Former seems more consistent with other `SERVICE_CHECK_*` options.